float32 linear
float32 angular
---
bool result
